drizzlepac/hlautils/svm_quality_analysis.py

```diff
@@ -318,29 +442,23 @@ def find_gaia_sources(hap_obj, log_level=logutil.logging.NOTSET):
         imgname = "{}_{}".format(parse_info[4], parse_info[5])
```

Comment:
Add a comment which contains an example of edp_item.info and hap_obj.info so the user can see what parse_info[4] and parse_info[5] are.

Reply:
Issue resolved by commit [3e10d25](https://github.com/spacetelescope/drizzlepac/pull/636/commits/3e10d2506cf7eca8c50d93f96b38456cc1fc8752)